### PR TITLE
Add more things on README.md

### PR DESCRIPTION
**Pull request by PatoFlamejanteTV**
Added some shields.io badges and some markdown editing on README.md, mainly editing markdown.

### CODE REVIEW

**Review by arduano — CHANGES_REQUESTED** [review]
I'd say replace the commit activity with a downloads counter please, the feeling of a dead repo is annoying to volunteer maintainers lol

**Comment by PatoFlamejanteTV** [comment]
ok

File: README.md [hunk 1]
```diff
@@ -1,30 +1,38 @@
 <h1 align="center">Wasabi</h1>
-<p align="center">Wasabi is a modern and fast real-time MIDI player written in Rust.</p>
 <p align="center"><img src="/assets/logo.svg" width="256"/></p>
+<p align="center">Wasabi is a modern and fast real-time MIDI player written in Rust.</p>
+<p align="center">
+<img alt="GitHub repo size" src="https://img.shields.io/github/repo-size/BlackMIDIDevs/wasabi">
+<img alt="GitHub License" src="https://img.shields.io/github/license/BlackMIDIDevs/wasabi">
+<img alt="GitHub Release" src="https://img.shields.io/github/v/release/BlackMIDIDevs/wasabi">
```
Comment (arduano) [inline README.md:7]:
```suggestion
<img alt="GitHub Release" src="https://img.shields.io/github/v/release/BlackMIDIDevs/wasabi">
<a href="https://github.com/BlackMIDIDevs/wasabi/releases/"><img src="https://img.shields.io/github/v/downloads/BlackMIDIDevs/wasabi"></a>
```

File: README.md [hunk 2]
```diff
@@ -1,30 +1,39 @@
...
+<a href="https://github.com/BlackMIDIDevs/wasabi/releases/"><img src="https://img.shields.io/github/v/downloads/BlackMIDIDevs/wasabi"></a>
```
Comment (arduano) [inline README.md:8]:
```suggestion
<a href="https://github.com/BlackMIDIDevs/wasabi/releases/"><img src="https://img.shields.io/github/downloads/BlackMIDIDevs/wasabi"></a>
```

I messed up whoops

**Comment by arduano** [comment]
Huh it's still not working

**Comment by arduano** [comment]
can you please make it work like in this repo? https://github.com/arduano/Zenith-MIDI

**Comment by PatoFlamejanteTV** [comment]
You mean the style (square)?

**Comment by arduano** [comment]
The downloads counter in general, the suggestion I added is broken
image

**Comment by PatoFlamejanteTV** [comment]
fixed!

**Review by arduano — APPROVED** [review]
Thanks!

**Pull request merged**